### PR TITLE
Fix/headerlogoimage

### PR DESCRIPTION
**Pull request by ronaklala**
image

### CODE REVIEW

File: components/Navbar/Navbar.js [hunk 1]
```diff
@@ -3,9 +3,15 @@ import Link from "next/link";
 const Navbar = () => {
   return (
     <nav className="navbar navbar-expand-lg  navbar-dark bg-dark ">
-      <div className="container-fluid">
+      <div className="container-fluid homebox">
         <a className="navbar-brand" href="#">
-          <Image src="/assets/images/logo.png" width={130} height={77} alt="" />{" "}
+          <img
+            src="https://pps.whatsapp.net/v/t61.24694-24/158318743_498252878509049_663621015264115443_n.jpg?ccb=11-4&oh=01_AVyllSGcdF9fqVxG50OVLNHY_49jamkcJdN0OBslszHErQ&oe=62D6BDB8"
```
Comment (ypdalvi) [inline components/Navbar/Navbar.js:9]: add image properly to assets folder

File: styles/globals.css [hunk 2]
```diff
@@ -137,3 +139,28 @@ ul {
 ul a {
   display: block;
 }
+
+.navbar-nav {
+  align-items: center !important;
```
Comment (ypdalvi) [inline styles/globals.css:144]: avoid using important everywhere

File: components/Navbar/Navbar.js (hunk 1 above)
Comment (ronaklala) [reply components/Navbar/Navbar.js:9]: It would be better if we call it from a CDN link, as it will make the page more light weight

Comment (ypdalvi) [reply components/Navbar/Navbar.js:9]: is the cdn link valid? this seems like a WhatsApp link 

Screenshot 2022-09-18 at 11 20 28 PM

**Pull request closed without merging**